lsst_efd_client/efd_helper.py

```diff
@@ -205,7 +212,8 @@ def build_time_range_query(self, topic_name, fields, start, end, is_window=False
                 start_str = start.isot
                 end_str = (start + end).isot
         elif isinstance(end, Time):
-            end = getattr(end, self.internal_scale)
+            if not end.scale == 'utc':
+                raise ValueError(f'Timestamps must be in UTC.')
```

Comment:
I see that you decided not to convert the `start` and `end` time scale anymore, and defer it to the user instead. I think it is fine.

Reply:
Yes.  I thought about issuing a warning as we did before, but now that we only allow one internal scale, it actually seemed more dangerous to do that than to except early and make the user do the conversion explicitly